py/README.rst

```diff
@@ -13,7 +13,7 @@ Usage
 To install it execute::
 
     $ python3.5 -m venv venv
-    $ venv/bin/python -m pip install -r requirements.txt
+    $ venv/bin/python -m pip install -r py/requirements.txt
```

Comment:
Not necessarily.. the instruction `python3.5 -m venv venv` will create a `venv` folder in the `current` directory